Sources/PackageCollections/Model/Package.swift

```diff
@@ -115,12 +128,9 @@ extension PackageCollectionsModel.Package {
 
         /// The package version's supported platforms
         public let minimumPlatformVersions: [SupportedPlatform]?
-
-        /// The package version's supported platforms verified to work
-        public let verifiedPlatforms: [PackageModel.Platform]?
-
-        /// The package version's Swift versions verified to work
-        public let verifiedSwiftVersions: [SwiftLanguageVersion]?
+        
+        /// An array of compatible platforms and Swift versions that has been tested and verified for.
+        public let verifiedCompatibility: [PackageCollectionsModel.Compatibility]?
```

Comment:
@neonichu Would it be better if we keep `verifiedPlatforms` and `verifiedSwiftVersions` temporarily?

Reply:
Keeping these for now https://github.com/apple/swift-package-manager/pull/3122/commits/3a7f17fa2c6fe00a74476061a5aac4aad22fa92e

Reply:
you can add a deprecated warning to make it clear that calcite needs to transition